### DISCUSSION TITLE
Internship Discussions!

### DISCUSSION THREAD

**Discussion by ghost** [jenndryden/Canadian-Tech-Internships-and-New-Grad-2025#164] [General]
Have you ever recieved a job offer through this repo??

**Comment by tohmze** [comment]
> Have you ever recieved a job offer through this repo??

I've gotten interviews 🤷🏾‍♂️

**Reply by ghost** [reply]
which company ??

**Reply by tohmze** [reply]
> which company ??

Goldman Sachs

**Reply by Khalid0md** [reply]
did you do the superday this past week?

**Reply by tohmze** [reply]
> did you do the superday this past week?

Yupp, what about you?

**Reply by Khalid0md** [reply]
yep

**Reply by tohmze** [reply]
> yep

Oh great! is there a way I can ask you some questions about your own experience?

**Reply by Khalid0md** [reply]
for sure, find me on linkedin, info is in my bio

**Reply by tohmze** [reply]
> for sure, find me on linkedin, info is in my bio

Sent an invitation

**Comment by ypatel2022** [comment]
Got 2 AMD interviews

**Comment by ypatel2022** [comment]
sadly i was not eligible 😭😭 (im in second year)

**Reply by ghost** [reply]
what do you think I am eligible for internship I am graduated.
